packages/catalog-process/src/routers/EServiceRouter.ts

```diff
@@ -174,6 +174,32 @@ const eservicesRouter = (
         }
       }
     )
+    .post(
+      "/eservices/templates/:templateId/instances",
+      authorizationMiddleware([ADMIN_ROLE, API_ROLE]),
+      async (req, res) => {
+        const ctx = fromAppContext(req.ctx);
+        try {
+          const eService =
+            await catalogService.createEServiceInstanceFromTemplate(
+              unsafeBrandId(req.params.templateId),
+              req.body,
+              ctx
+            );
+          return res
+            .status(200)
+            .send(catalogApi.EService.parse(eServiceToApiEService(eService)));
+        } catch (error) {
+          const errorRes = makeApiProblem(
+            error,
+            createEServiceErrorMapper,
```

Comment:
There should be a custom error mapper that includes the instance creation related error

Reply:
Yes, you'll need to create one specifically for this route